.github/workflows/nightly.yml

```diff
@@ -99,5 +99,5 @@ jobs:
         VSCODE_SWIFT_VSIX_PRERELEASE=1
         GITHUB_REPOSITORY=${{github.repository}}
       windows_pre_build_command: . .github\workflows\scripts\windows\setup.ps1
-      windows_build_command: scripts\test_windows.ps1
+      windows_build_command: .github/workflows/nightly.yml scripts\test_windows.ps1
```

Comment:
This looks odd... invoking a yaml file with a powershell script as an argument? how will that work?

Reply:
Sorry, did these quickly 😅 Too much copy paste